src/sentry/models/authprovider.py

```diff
@@ -119,8 +119,7 @@ def enable_scim(self, user):
                 "member:write",
                 "member:admin",
                 "team:write",
-                "org:write",
-                "org:admin",
+                "team:admin",
```

Comment:
updated permissions for the SCIM api key to align with how permissions are set on the routes.